src/Objs/SearchResults/SearchInput.js

```diff
@@ -10,10 +10,13 @@ class SearchInput extends React.Component {
     };
   }
 
-  componentWillReceiveProps(nextProps) {
-    if (this.props.params.q !== nextProps.params.q) {
-      this.setState({ q: nextProps.params.q });
+  static getDerivedStateFromProps(nextProps, prevState) {
+    if (!prevState.params || !nextProps.params || prevState.params.q === nextProps.params.q) {
```

Comment:
* [ ] I don't think the change in behavior is intentional: It is no longer possible to clear `state.q` by passing no `params`.